examples/performance/01_gpu.ipynb

```diff
@@ -137,17 +137,13 @@
    "cell_type": "code",
```

Comment:
why is this notebook changing all of a sudden?

Reply:
The generated code has changed for the current version including `device(devicenum)`